.github/workflows/release.yml

```diff
@@ -7,8 +7,6 @@ on:
 env:
```

Comment:
do we need this at all anymore now that we won't be releasing from this repo? or will we be releasing docs with this?

Reply:
We'll be releasing docs from this repo, but what that ultimately looks like is still WIP.